1.6/Dockerfile.rhel7

```diff
@@ -46,9 +45,11 @@ RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
     mv oc /usr/local/bin && \
     rm oadm && \
     rm openshift && \
-    chmod -R og+rw /opt/openshift
+    mkdir -p /opt/openshift && chmod -R og+rw /opt/openshift && \
+    mkdir -p /usr/share/jenkins && chmod -R og+rw /usr/share/jenkins
 
-VOLUME ["/var/lib/jenkins"]
+COPY ./contrib/openshift /opt/openshift
+COPY ./contrib/jenkins /usr/local/bin
```

Comment:
Permissions in the original image:

```
$ ls -la /opt/openshift/
total 16
drwxrwxrwx. 3 root root 4096 Aug 27 13:10 .
drwxr-xr-x. 3 root root 4096 Aug 27 13:10 ..
drwxrwxrwx. 2 root root 4096 Aug 27 13:20 configuration
-rwxrwxrwx. 1 root root 2039 Aug 27  2015 password-encoder.jar
```

With COPY after RUN:

```
$ ls -la /opt/openshift/
total 16
drwxrwxrwx. 3 root root 4096 Aug 27 13:42 .
drwxr-xr-x. 3 root root 4096 Aug 27 13:41 ..
drwxrwxr-x. 4 root root 4096 Aug 27 13:22 configuration
-rwxrwxr-x. 1 root root 2039 Aug 27 13:22 password-encoder.jar
```

And I was able to, as UID 1001, make changes to `/opt/openshift/configuration/config.xml`.

What exactly could go problematic here?